.github/workflows/promote.yml

```diff
@@ -0,0 +1,69 @@
+name: Promote Edge builds to Stable
+
+on:
+  workflow_dispatch:
+  workflow_call:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}-promote
+  cancel-in-progress: true
+
+jobs:
+  promote:
+    runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: false
+      matrix:
+        image_name:
+          - bazzite
+          - bazzite-nvidia
+          - bazzite-nvidia-open
+          - bazzite-gnome
+          - bazzite-gnome-nvidia
+          - bazzite-gnome-nvidia-open
+          - bazzite-deck
+          - bazzite-deck-gnome
+          - bazzite-ally
+          - bazzite-ally-gnome
+          - bazzite-asus
+          - bazzite-gnome-asus
+          - bazzite-asus-nvidia
+          - bazzite-gnome-asus-nvidia
+          - bazzite-asus-nvidia-open
+          - bazzite-gnome-asus-nvidia-open
+        major_version: [40]
+    steps:
+
+      # Workaround bug where capital letters in your GitHub username make it impossible to push to GHCR.
+      # https://github.com/macbre/push-to-ghcr/issues/12
+      - name: Lowercase Registry
+        id: registry_case
+        uses: ASzc/change-string-case-action@v6
+        with:
+          string: ${{ env.IMAGE_REGISTRY }}
+
+      - name: Pull Image (Edge tag)
+        uses: Wandalen/wretry.action@v3.5.0
+        id: pull
+        with:
```

Comment:
Do we need to pull here?  Skopeo can add tags to images in a remote registry without pulling.

Reply:
Give me the command and I will replace it. Am not aware of it.

Reply:
I'm on mobile, so can't make something that's drop-in, but this is how I've done it in the past

```
            skopeo copy docker://${IMAGE_REGISTRY}@${IMAGE_DIGEST} docker://${IMAGE_REGISTRY}:latest
```

The digest can be replaced with an existing tag